testing/jormungandr-scenario-tests/src/scenario/repository/mod.rs

```diff
@@ -176,6 +176,12 @@ fn scenarios_repository() -> Vec<Scenario> {
             transaction_to_passive,
             vec![Tag::Short],
         ),
+        #[cfg(feature = "desync-tests")]
```

Comment:
I think adding another tag: Tag::FakeTime should solve the issue.  So no need to add another cfg, since it looks like this is runtime dependency

Reply:
I know that i was proposing that, but after looking at the code it should be able to solve with separate Tag